lib/facter/facts_utils/virtual_detector.rb

```diff
@@ -8,7 +8,8 @@ def initialize
 
     def platform # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
       fact_value = check_docker_lxc || check_dmi || check_gce || retrieve_from_virt_what || check_vmware
-      fact_value ||= check_open_vz || check_vserver || check_xen || check_other_facts || check_lspci || 'physical'
+      fact_value ||= check_jail || check_open_vz || check_vserver || check_xen || check_other_facts
+      fact_value ||= check_lspci || check_pciconf || 'physical'
```

Comment:
`check_jail` and `check_pciconf` where added to this list.  I attempted to keep the names sorted by "type" and then alphabetically, but I feel like the beginning of the list makes no sense… Please tell me how to order these if you think of a better arrangement.